data/clusterPacks.yaml

```diff
@@ -13,12 +13,12 @@
       single node.  
     - Operates the [Slurm Workload Manager](https://slurm.schedmd.com/) job
       scheduler.  
-    - 2 [Alces Flight Center](https://center.alces-flight.com) support credits
+    - 2 [ConcertIM](https://portal.concertim.com) support credits
       per month.
   details: |
     A cluster comprising an entry-level login node and general-level compute
     nodes (3.75 GiB, 2 CPUs) provided for familiarization and experimentation
-    with Alces Flight Compute. By default, the compute estate runs on two
+    with ConcertIM Compute. By default, the compute estate runs on two
```

Comment:
No, this one really is [Alces Flight Compute](https://alces-flight-appliance-docs.readthedocs.io/en/stable/overview/whatisit.html) unless there's more I haven't been told...